clannetwork/chain.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "clannetwork",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Clan Network",
+  "chain_id": "playstation-2",
+  "bech32_prefix": "clan",
```

Comment:
Please register this bech32 prefix to SLIP173:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md